<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoBike Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <!-- ========== STATION TAB ========== -->
      <widget class="QWidget" name="station_tab">
       <attribute name="title">
        <string>Station</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_station">
        <item row="0" column="0">
         <widget class="QLabel" name="label_stn_title">
          <property name="text">
           <string>Add Station Info</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_stn_name">
          <property name="text">
           <string>Station Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="stn_name_text"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_stn_lat">
          <property name="text">
           <string>Latitude:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="stn_ltd_text"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_stn_long">
          <property name="text">
           <string>Longitude:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="stn_lng_text"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="stn_add_btn">
          <property name="text">
           <string>Add Station</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- ========== TIME TAB ========== -->
      <widget class="QWidget" name="time_tab">
       <attribute name="title">
        <string>Time</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_time">
        <item row="0" column="0">
         <widget class="QLabel" name="label_tme_title">
          <property name="text">
           <string>Add Time Info</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0"><widget class="QLabel" name="label_date"><property name="text"><string>Date:</string></property></widget></item>
        <item row="1" column="1"><widget class="QLineEdit" name="tme_date_text"/></item>
        <item row="2" column="0"><widget class="QLabel" name="label_hour"><property name="text"><string>Hour:</string></property></widget></item>
        <item row="2" column="1"><widget class="QLineEdit" name="tme_hour_text"/></item>
        <item row="3" column="0"><widget class="QLabel" name="label_day"><property name="text"><string>Day:</string></property></widget></item>
        <item row="3" column="1"><widget class="QLineEdit" name="tme_day_text"/></item>
        <item row="4" column="0"><widget class="QLabel" name="label_month"><property name="text"><string>Month:</string></property></widget></item>
        <item row="4" column="1"><widget class="QLineEdit" name="tme_month_text"/></item>
        <item row="5" column="0"><widget class="QLabel" name="label_year"><property name="text"><string>Year:</string></property></widget></item>
        <item row="5" column="1"><widget class="QLineEdit" name="tme_year_text"/></item>
        <item row="6" column="0"><widget class="QLabel" name="label_day_of_week"><property name="text"><string>Day of Week:</string></property></widget></item>
        <item row="6" column="1"><widget class="QLineEdit" name="tme_day_of_week_text"/></item>
        <item row="7" column="1"><widget class="QPushButton" name="tme_add_btn"><property name="text"><string>Add Time</string></property></widget></item>
       </layout>
      </widget>

      <!-- ========== TRIP TAB ========== -->
      <widget class="QWidget" name="trip_tab">
       <attribute name="title">
        <string>Trip</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_trip">
        <item row="0" column="0"><widget class="QLabel" name="label_trp_title"><property name="text"><string>Add Trip Info</string></property></widget></item>
        <item row="1" column="0"><widget class="QLabel" name="label_start_time"><property name="text"><string>Start Time:</string></property></widget></item>
        <item row="1" column="1"><widget class="QLineEdit" name="trp_start_time_text"/></item>
        <item row="2" column="0"><widget class="QLabel" name="label_end_time"><property name="text"><string>End Time:</string></property></widget></item>
        <item row="2" column="1"><widget class="QLineEdit" name="trp_end_time_text"/></item>
        <item row="3" column="0"><widget class="QLabel" name="label_duration"><property name="text"><string>Duration (sec):</string></property></widget></item>
        <item row="3" column="1"><widget class="QLineEdit" name="trp_duration_sec_text"/></item>
        <item row="4" column="0"><widget class="QLabel" name="label_start_stn"><property name="text"><string>Start Station ID:</string></property></widget></item>
        <item row="4" column="1"><widget class="QLineEdit" name="trp_start_stn_id_text"/></item>
        <item row="5" column="0"><widget class="QLabel" name="label_end_stn"><property name="text"><string>End Station ID:</string></property></widget></item>
        <item row="5" column="1"><widget class="QLineEdit" name="trp_end_stn_id_text"/></item>
        <item row="6" column="0"><widget class="QLabel" name="label_user_id"><property name="text"><string>User ID:</string></property></widget></item>
        <item row="6" column="1"><widget class="QLineEdit" name="trp_user_id_text"/></item>
        <item row="7" column="0"><widget class="QLabel" name="label_time_id"><property name="text"><string>Time ID:</string></property></widget></item>
        <item row="7" column="1"><widget class="QLineEdit" name="trp_time_id_text"/></item>
        <item row="8" column="1"><widget class="QPushButton" name="trp_add_btn"><property name="text"><string>Add Trip</string></property></widget></item>
       </layout>
      </widget>

      <!-- ========== USER TAB ========== -->
      <widget class="QWidget" name="user_tab">
       <attribute name="title">
        <string>User</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_user">
        <item row="0" column="0"><widget class="QLabel" name="label_usr_title"><property name="text"><string>Add User Info</string></property></widget></item>
        <item row="1" column="0"><widget class="QLabel" name="label_usr_name"><property name="text"><string>User Name:</string></property></widget></item>
        <item row="1" column="1"><widget class="QLineEdit" name="usr_name_text"/></item>
        <item row="2" column="0"><widget class="QLabel" name="label_usr_age"><property name="text"><string>Age:</string></property></widget></item>
        <item row="2" column="1"><widget class="QLineEdit" name="usr_age_text"/></item>
        <item row="3" column="0"><widget class="QLabel" name="label_usr_gender"><property name="text"><string>Gender:</string></property></widget></item>
        <item row="3" column="1"><widget class="QLineEdit" name="usr_gender_text"/></item>
        <item row="4" column="0"><widget class="QLabel" name="label_usr_type"><property name="text"><string>User Type:</string></property></widget></item>
        <item row="4" column="1"><widget class="QLineEdit" name="usr_type_text"/></item>
        <item row="5" column="1"><widget class="QPushButton" name="usr_add_btn"><property name="text"><string>Add User</string></property></widget></item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
